pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -585,82 +609,53 @@ func (r *Reconciler) runReconcileSeedFlow(
 				return err
 			}
 
-			filters.WriteString(fmt.Sprintln(loggingConfig.Filters))
-			parsers.WriteString(fmt.Sprintln(loggingConfig.Parsers))
+			if len(loggingConfig.Inputs) > 0 {
+				inputs = append(inputs, loggingConfig.Inputs...)
+			}
 
-			if loggingConfig.UserExposed {
-				userAllowedComponents = append(userAllowedComponents, loggingConfig.PodPrefixes...)
+			if len(loggingConfig.Filters) > 0 {
+				filters = append(filters, loggingConfig.Filters...)
 			}
-		}
 
-		loggingRewriteTagFilter := `[FILTER]
-    Name          modify
-    Match         kubernetes.*
-    Condition     Key_value_matches tag ^kubernetes\.var\.log\.containers\.(` + strings.Join(userAllowedComponents, "|") + `)-.+?_
-    Add           __gardener_multitenant_id__ operator;user
```

Comment:
Hm, @vlvasilev can't this `Add           __gardener_multitenant_id__ operator;user` part be removed after https://github.com/gardener/gardener/pull/7523?